duckduckgo/Android
# Live update Privacy Dashboard content

Asana Issue URL: https://app.asana.com/0/361428290920652/488066724028605

## Description
Updates the Privacy Dashboard so that it refreshes as a page loads. Includes refactoring to remove duplicate tracker network look ups as these were slow.

## Steps to Test this PR:
1. Navigate to a url with many trackers e.g cnn.com
1. While cnn is loading open the Privacy Dashboard
1. Note that the Privacy Dashboard page updates as the webpage loads
